src/modules/displayCards.js

```diff
@@ -31,6 +33,7 @@ const displayCards = async () => {
   </div>
     `;
     main.appendChild(card);
+    counters();
```

Comment:
The counter doesn't need to run everytime you append a card to the main. We can run the counters() function after the forEach loop so we only count once after all of the appending has finished